lib/learn_submit/version.rb

```diff
@@ -1,3 +1,3 @@
 module LearnSubmit
-  VERSION = '1.3.0'
+  VERSION = '1.3.1'
```

Comment:
@gj just a note, i find it preferable to wait to bump the version until PRs are merged to master and then doing a single commit that just does the version bump and tagging that commit as a release. it's not a big deal here since it's such a small change and there are no more incoming PRs. for example, on learn-test, we're putting your mocha stuff and the optimizations into the same release, if the first PR merged has the version bump and then another PR is merged in and maybe some fixes based off testing come in, and then we release that version it makes the timeline events a little weird because the version bump will have happened before a bunch of other commits in the history even though those later commits are included in the version.

Reply:
Makes total sense @joshrowley ! Will do going forward.